pipelines/update_database.py

```diff
@@ -0,0 +1,103 @@
+import os
+from pathlib import Path
+
+import geopandas as gpd
+import pandas as pd
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+
+from src.constants import ISO3S
+from src.datasources import codab, floodscan
+from src.utils import blob
+
+load_dotenv()
+
+AZURE_DB_PW_DEV = os.getenv("AZURE_DB_PW_DEV")
+AZURE_DB_UID = os.getenv("AZURE_DB_UID")
+LOCAL_DATA_DIR = Path("pipelines/data")
+
+
+def migrate_data_to_repo(clobber: bool = False):
+    save_path = LOCAL_DATA_DIR / "adm.shp"
```

Comment:
I think we'd either need to un-`gitignore` the `data` directory, or add something like:
```
if not LOCAL_DATA_DIR.exists():
    LOCAL_DATA_DIR.mkdir(parents=True)
```
just for this to work.

Or, we can forgo the local CODAB storage altogether, and just `.load_codab_from_blob()` each time.